### DISCUSSION TITLE
Template of templates

### DISCUSSION THREAD

**Discussion by sigord** [SilentVoid13/Templater#1111] [Help]
Hello everyone!

I found [THIS](https://github.com/SilentVoid13/Templater/discussions/259) template useful for my purposes. Many thanks to @megclaypool

I need to create many castom templates from it. And I decide to make a template of templates, but I don't know how to do this.

**Comment by megclaypool** [comment]
Hmmm... I'm not quite sure what you're trying to do with a template of templates. 

Maybe this would be helpful: you can call one template from inside another: `<% tp.file.include("[[NAME OF YOUR TEMPLATE GOES HERE]]") %>`.  (The inner bit is just a regular obsidian link, so if you're having trouble getting it working, create it in any regular obsidian file and then copy it into your template 🙂)

I use it to include a timeline in both my daily and weekly templates, but the sky's the limit 🙂
